docs/src/languages/nodejs/debug.md

```diff
@@ -3,121 +3,244 @@ title: "Debugging"
 For help, see: https://nodejs.org/en/docs/inspector
 Listening on port 8888
 ```
 
-### Forward the debugger port locally
+<--->
 
-In another terminal, create an SSH tunnel that forwards the 9229 port:
+---
+title=Local
+file=none
+highlight=false
+---
 
-```sh
-$ ssh -N -L9229:127.0.0.1:9229  $(platform  ssh --pipe)
+In the same terminal as the previous step, run the following command:
+
+```bash
+node --inspect <START_FILE>
 ```
 
-### Connect the debugger
+Replace `<START_FILE>` with the file defined for [your app's `start` command](./_index.md#usage-example).
+<!--vale off -->
+{{< /codetabs >}}
+<!--vale on -->
 
-You can now connect the debugger as if you were debugging a local application.
+{{< codetabs >}}
+
+---
+title=Remote
+file=none
+highlight=false
+---
 
-#### Using the chrome devtools
+Stop the current process and restart it in inspect mode:
 
-Go to `chrome://inspect`. You should see the running application under the `Remote Target` list and be able to click the `inspect` link to open the debugger.
+```bash
+sv stop app
+node --inspect <START_FILE>
+```
 
-#### Using VSCode
+Replace `<START_FILE>` with the file defined for [your app's `start` command](./_index.md#4-start-your-app).
 
-Use the `Node.js: Attach` debugger option. If not configured already, go to the debugger tab and select `create a launch.json file` and select `Node.js`. Then use intellisense (usually `control`+`space`) in the `configurations` array. Select `Node.js: Attach`. You can now select `Attach` in the `RUN AND DEBUG` drop-down and click the green arrow. You will find the JS files in which to set the breakpoints in the `LOADED SCRIPTS` view, under `Attach: Remote Process [0]` -> `/app`.
+<--->
 
-## Locally
+---
+title=Local
+file=none
+highlight=false
+---
 
-### Create a new environment
+In the same terminal as the previous step, run the following command:
 
-```sh
-$ platform branch debug-branch
+```bash
+node --inspect <START_FILE>
 ```
 
-### Open a tunnel
+Replace `<START_FILE>` with the file defined for [your app's `start` command](./_index.md#usage-example).
+<!--vale off -->
+{{< /codetabs >}}
+<!--vale on -->
+
 
-To get access to your application's relationships locally, open a tunnel to your application container:
+You get output something like this:
 
-```sh
-$ platform tunnel:open
+```bash
+Debugger listening on ws://127.0.0.1:9229/10701e5d-d627-4180-a967-d47a924c93c0
+For help, see: https://nodejs.org/en/docs/inspector
+Listening on port 8888
 ```
 
-In a multi-application project, you will be prompted to chose which application to open a tunnel to. You can also specifiy the application you want by using the `--app` option:
+## 4. (If debugging remotely) Forward the debugger port locally
 
-```sh
-$ platform tunnel:open --app=my-app-name
+In another terminal, create an SSH tunnel that forwards to the 9229 port:
+
+```bash
+ssh -N -L9229:127.0.0.1:9229  $(platform  ssh --pipe)
```

Comment:
:hammer_and_wrench:
To make the example more readable and re-usable, a bit more spacing and using `localhost` instead of `127.0.0.1` should lead to [the same behavior](https://www.wikiwand.com/en/Hosts_(file)#/File_content).
```suggestion
ssh -N -L 9229:localhost:9229 $(platform ssh --pipe)
```